### PR TITLE
Bump cadvisor godep

### LINKED ISSUE #13189: Zero network traffic from Kubelet stats/metrics on docker >= 1.7
Bug is in cadvisor, fix is merged - will submit a PR to bump cadvisor dependency.

### PR DESCRIPTION
**Pull request by jimmidyson**
Fixes #13189, Fixes kubernetes/heapster#500

### CODE REVIEW

**Comment by jimmidyson** [comment]
@vishh - tested successfully that the updated cadvisor works with kubernetes.

**Comment by vishh** [comment]
ok to test

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "go1.4",
```
Comment (vishh) [inline Godeps/Godeps.json]: Why change the go version?

**Comment by vishh** [comment]
@jimmidyson: Thanks for fixing the networking issue. Except for the one comment, this PR LGTM.

File: Godeps/Godeps.json (hunk 1 above)
Comment (jimmidyson) [inline Godeps/Godeps.json]: Oops - fixed.

**Comment by vishh** [comment]
LGTM.

**Pull request merged**